keep/api/models/db/topology.py

```diff
@@ -12,10 +12,16 @@ class TopologyServiceApplication(SQLModel, table=True):
     application_id: UUID = Field(foreign_key="topologyapplication.id", primary_key=True)
 
     service: "TopologyService" = Relationship(
-        sa_relationship_kwargs={"primaryjoin": "TopologyService.id == TopologyServiceApplication.service_id"},
+        sa_relationship_kwargs={
+            "primaryjoin": "TopologyService.id == TopologyServiceApplication.service_id",
+            "viewonly": "True",
```

Comment:
what does that do? why is it changing here?

Reply:
Two write-allowed relations to the same model causes minor conflicts in sqlalchemy. Make one of them viewonly resolves this. No affect on our logic, since we are not using this relations to add items